src/Relation/Binary/Lattice/Properties/BoundedMeetSemilattice.agda

```diff
@@ -14,6 +14,7 @@ module Relation.Binary.Lattice.Properties.BoundedMeetSemilattice
 open BoundedMeetSemilattice M
 
 open import Algebra.Definitions _≈_
+open import Algebra.Ordered.Bundles using (CommutativePomonoid)
```

Comment:
Err this import doesn't seem to be used? Looks like you are missing adding something to this file?

Reply:
Huh, probably redundant, yes. There was an extra proof earlier, but it was a "duplicated" solved via a re-export. Will double-check.